src/plugins/hookspec.py

```diff
@@ -295,6 +295,12 @@ def get_tournament_tie_breaks_warning_message(
     ) -> str | None:
         """Warning message for the tie-breaks of a tournament."""
 
+    @hookspec(firstresult=True)
+    def get_tournament_pairing_settings_warning_message(
```

Comment:
```suggestion
    def get_tournament_pairing_warning_message(
```
the warning is displayed on the pairing part of the card, it has nothing to do with the settings (to replace in other places too)